base_model/lasso.R

```diff
@@ -0,0 +1,55 @@
+library(glmnet) # load 'glmnet' package
+
+data <- readRDS("./Data/df_index_demographic_vars.RDS") # read index and demographics data provided by Carolina
+
+data <- data[,-1] # remove 'control', an ID variable
+data$sex <- data$sex-1 # make 'sex' a 0/1 binary variable
+data$data_year <- data$data_year-1997 # normalize 'data_year' to start at 1997
```

Comment:
The data years are 1997-2013, every two years. Does this mean that the possible values for year are: 0 (1997), 2 (1999), 4 (2001), and so on?

Reply:
That is correct! 1997 would be year 0, 1999 would be year 2, etc.